app/models/manageiq/providers/amazon/inventory_collection_default/cloud_manager.rb

```diff
@@ -190,11 +190,13 @@ def vm_and_template_taggings(extra_attributes = {})
       }
 
       attributes[:targeted_arel] = lambda do |inventory_collection|
-        ids = inventory_collection.parent_inventory_collections.collect_concat(&:id)
+        manager_uuids = inventory_collection.parent_inventory_collections.collect(&:manager_uuids).map(&:to_a).flatten
+        ems = inventory_collection.parent
         Tagging.where(
           'taggable_type' => 'VmOrTemplate',
-          'taggable_id'   => ids
-        ).joins(:tag).merge(Tag.controlled_by_mapping)
+          'taggable_id'   => ems.vm_and_template_taggings.where(:ems_ref => manager_uuids)
```

Comment:
just 
```
ems.vm_and_templates.where(:ems_ref => manager_uuids)
```

since the manager uuis are fro ma table VmOrTemplate

Reply:
thanks, fixed